holodev

```diff
@@ -167,6 +163,11 @@ setup_libvirt_Arch() {
   if [ ! -f /etc/lxc/default.conf.bkp ]; then
     sudo cp /etc/lxc/default.conf{,.bkp}
   fi
+sudo sh -c 'cat << EOF > /etc/lxc/default.conf
```

Comment:
Are you sure this necessary? Part of these options already added in container config file, see holodev function create_default_configuration_file().